cmd/crane/cmd/mutate.go

```diff
@@ -143,6 +148,7 @@ func NewCmdMutate(options *[]crane.Option) *cobra.Command {
 	mutateCmd.Flags().StringSliceVar(&cmd, "cmd", nil, "New cmd to set")
 	mutateCmd.Flags().StringVarP(&newRef, "tag", "t", "", "New tag to apply to mutated image. If not provided, push by digest to the original image repository.")
 	mutateCmd.Flags().StringSliceVar(&newLayers, "append", []string{}, "Path to tarball to append to image")
+	mutateCmd.Flags().StringVar(&workingDir, "working_dir", "", "New WorkingDir to set")
```

Comment:
I'd make this flag `--workdir` to match Dockerfile's `WORKDIR` mostly because I don't like underscores in flags... @imjasonh any strong opinions?

Reply:
I hate all the possible options roughly equally.

I wish Cobra supported flag aliases better, I'd want all of `--workingdir`, `--working_dir`, `--working-dir`, `--workdir`, etc., to map to the same value.

Given the options I think I actually prefer `--working_dir` over `--workdir`, since that's what it's called in the config, and `WORKDIR` seems to only be a Dockerfileism.

Reply:
Changed to --working_dir